docs/robots/dingo/tutorials_dingo.mdx

```diff
@@ -4,7 +4,1023 @@ sidebar_position: 4
+
+    :::note
+    The **Drive Test** drives the robot using the Odometry data, which inherently will not be 100% accurate. Therefore, some undershoot/overshoot
+    is to be expected.
+    :::
+
+- **Cooling Test**
+
+    The **Cooling Test** is an optional test that only applies to Dingo's with an external fan connected to the MCU, and checks that the external
+    fan is working properly.
+
+    This test makes the fan spin at different speeds by publishing fan speed commands to the `/mcu/cmd_fans` ROS topic. The user will be asked
+    to verify that the fan change to the expected speeds.
+
+### CAN Bus Test
+
+The `check_can_bus_interface` script checks that communication between the motors, encoders, robot's MCU, and robot's computer are working
+properly over the CAN bus interface.
+
+This script verifies that the `can0` interface is detected and activated, then proceeds to check the output of `candump` to verify that good
+CAN packets are being transmitted. Based on the Dingo configuration, either Dingo-D or Dingo-O, this script will know to check for good CAN
+packets from 2 or 4 encoders, respectively.
+
+#### Running CAN Bus Test
+
+To run the `check_can_bus_interface` script on a Dingo robot, open terminal and run:
+
+```bash
+rosrun dingo_tests check_can_bus_interface
+```
+
+---
+
+## Advanced Topics
+
+### Configuring the Network Bridge
+
+<ComponentConfiguringNetworkBridge />
+
+### Jetson Installation
+
+Refer to the [Jetson](/docs/computers/jetson) page for details on installing a Jetson in Dingo.
+
+### Additional Simulation Worlds    {#additional-sim}
+
+In addtion to the default `dingo_world.launch` file, `dingo_gazebo` contains two additional launch files:
+- `empty_world.launch`, which spawns Dingo in a featureless, infinite plane;
+- `spawn_dingo.launch`, which is intended to be included in any custom world to add a Dingo simulation to it.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/dingo_images/dingo_empty_world.png"
+      width="800"
+    />
+    <figcaption>Dingo in the Empty World environment</figcaption>
+  </figure>
+</center>
+
+To add a Dingo to any of your own worlds, simply include the `spawn_dingo.launch` file in your own world's launch:
+
+```markup
+<include file="$(find dingo_gazebo)/launch/spawn_dingo.launch">
+  <!-- Optionally configure the spawn position -->
+  <arg name="x" value="$(arg x)"/>
+  <arg name="y" value="$(arg y)"/>
+  <arg name="z" value="$(arg z)"/>
+  <arg name="yaw" value="$(arg yaw)"/>
+</include>
+```
+
+Finally, Clearpath provides an additional suite of simulation environments that can be downloaded separately and used
+with Dingo, as described below.
+
+#### Clearpath Gazebo Worlds
+
+The Clearpath Gazebo Worlds collection contains 4 different simulation worlds, representative of different
+environments our robots are designed to operate in:
+
+- Inspection World: a hilly outdoor world with water and a cave
```

Comment:
It seems odd to mention outdoor simulations for Dingo. Are these simulated worlds useful for a Dingo user?